libs/design/src/molecules/media-gallery/registry/media-gallery.registry.ts

```diff
@@ -1,72 +1,68 @@
 import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
 import { DaffMediaGalleryComponent } from '../media-gallery.component';
 import { DaffThumbnailDirective } from '../thumbnail/thumbnail.directive';
-import { BehaviorSubject, Observable } from 'rxjs';
 
 export interface DaffMediaGalleryDict {
```

Comment:
I changed this to a dictionary of behavior subjects rather than a behavior subject of a dictionary, so that each media-gallery-renderer instance doesn't need to subscribe to the changes of every media gallery.